src/rudderstack.ts

```diff
@@ -105,42 +105,34 @@ export class RudderStack {
   }
 
   init() {
-    const isProduction = process.env.CIRCLE_JOB === "release_production";
-    const isStaging = process.env.CIRCLE_JOB === "release_staging";
-
-    let RUDDERSTACK_KEY;
-    if (isProduction) {
-      RUDDERSTACK_KEY = process.env.RUDDERSTACK_PRODUCTION_KEY;
-    } else if (isStaging) {
-      RUDDERSTACK_KEY = process.env.RUDDERSTACK_STAGING_KEY;
-    }
+    const is_production = process.env.CIRCLE_JOB === "release_production";
 
+    const RUDDERSTACK_KEY = is_production
+        ? process.env.RUDDERSTACK_PRODUCTION_KEY
+        : process.env.RUDDERSTACK_STAGING_KEY;
     const RUDDERSTACK_URL = process.env.RUDDERSTACK_URL;
+
     if (RUDDERSTACK_KEY && RUDDERSTACK_URL) {
-      RudderAnalytics.load(RUDDERSTACK_KEY, RUDDERSTACK_URL);
-      RudderAnalytics.ready(() => {
-        this.has_initialized = true;
-      });
+        RudderAnalytics.load(RUDDERSTACK_KEY, RUDDERSTACK_URL);
+        RudderAnalytics.ready(() => {
+            this.has_initialized = true;
+        });
     }
   }
 
   identifyEvent = (user_id: string, payload: TEvents["identify"]) => {
     if (this.has_initialized) {
-      RudderAnalytics.identify(user_id, payload);
-      this.has_identified = true;
+        RudderAnalytics.identify(user_id, payload);
+        this.has_identified = true;
     }
   };
 
   /**
    * Pushes page view track event to rudderstack
    */
-  pageView(current_page: string) {
-    if (
-      this.has_initialized &&
-      this.has_identified &&
-      current_page !== this.current_page
-    ) {
-      RudderAnalytics.page("Deriv App", current_page);
+  pageView(current_page: string, platform: string = "Deriv App") {
+    if (this.has_initialized && this.has_identified && current_page !== this.current_page) {
+      RudderAnalytics.page(platform, current_page);
```

Comment:
The platform should be made as an argument, since now the rudderstack package can be used within deriv com